osm2lanes/src/road/separator.rs

```diff
@@ -4,6 +4,20 @@ use super::Printable;
 use crate::locale::Locale;
 use crate::metric::Metre;
 
+#[non_exhaustive]
+#[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum Semantic {
```

Comment:
Maybe after we settle on including this in the output, it would make sense to use the same enum internally and in the public API. But no need to rush that decision.

I also remembered https://wiki.openstreetmap.org/wiki/Proposed_features/cycleway:separation#Typical_values for future ideas here